<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GongJianBo1992</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="loadLayout">
      <item>
       <widget class="QPushButton" name="btnLoadPath">
        <property name="text">
         <string>loadpath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editLoadPath">
        <property name="text">
         <string>://simple_data.json</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadJson">
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="dumpLayout">
      <item>
       <widget class="QPushButton" name="btnDumpPath">
        <property name="text">
         <string>dumppath</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editDumpPath">
        <property name="text">
         <string>./dump_data.json</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDumpJson">
        <property name="text">
         <string>dump</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="actionLayout">
      <item>
       <widget class="QPushButton" name="btnInsert">
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnInsertChild">
        <property name="text">
         <string>InsertChild</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
